R/ds_search_sql.R

```diff
@@ -12,8 +12,10 @@
 #' ds_search_sql(sql2, url=url, as="table")
 #' }
 
-ds_search_sql <- function(sql, url = 'http://demo.ckan.org', as='list', ...) {
-  res <- POST(file.path(url, '/api/action/datastore_search_sql'), ctj(), query = list(sql = sql), ...)
+ds_search_sql <- function(sql, url=get_default_url(), as='list', ...) {
+  res <- POST(file.path(url, '/api/action/datastore_search_sql'),
+              ctj(),
+              query=list(sql = sql), ...)
   res <- content(res, "text")
-  switch(as, json = res, list = jsl(res), table = jsd(res))
+  switch(as, json=res, list=jsl(res), table=jsd(res))
```

Comment:
Yelp... OK I'l bring them back! I went with the Google R style guide which finds [spaces being optional for function parameters](https://google-styleguide.googlecode.com/svn/trunk/Rguide.xml#spacing). Agree with Hadley's "always spaces around infix operators".